engine/runtime-test-instruments/src/main/java/org/enso/interpreter/test/instruments/CodeIdsTestInstrument.java

```diff
@@ -21,8 +21,24 @@
     services = CodeIdsTestInstrument.class)
 public class CodeIdsTestInstrument extends TruffleInstrument {
```

Comment:
All you need to do here is to register the instruments, so Truffle finds them in a separate module.